content/en/blog/2019/multi-cluster-mesh-automation/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Multi-mesh configuration and service discovery using Admiral
+subtitle: Configuration automation for Istio multi-mesh.
+description: Automating Istio configuration for Istio deployments (clusters) that work as a single mesh.
+publishdate: 2019-12-28
+attribution: Anil Attuluri (Intuit), Jason Webb (Intuit)
+keywords: [traffic-management,automation,configuration,multicluster,multi-mesh,gateway,federated,globalidentifer]
+target_release: 1.5
+---
+
+When we read the blog post [Multi-Mesh Deployments for Isolation and Boundary Protection](https://istio.io/blog/2019/isolated-clusters/) we could immediately relate to some of the problems mentioned and how we are using [Admiral](https://github.com/istio-ecosystem/admiral), an open source project under istio-ecosystem to solve these.
```

Comment:
mentioned and... -> mentioned, and we thought we could share how we are using ... to solve similar issues.